frontend/src/pages/staffform/BcRegisteredClientInformationWizardStep.vue

```diff
@@ -0,0 +1,485 @@
+      :init-fetch="false"
+      :disabled="!formData.businessInformation.businessName"
+      #="{ content, loading, error }"
+    >
+      <AutoCompleteInputComponent
+        id="businessName"
+        label="Client name"
+        autocomplete="off"
+        required
+        required-label
+        tip=""
+        v-model="formData.businessInformation.businessName"
+        :contents="content"
+        :validations="[
+          ...getValidations('businessInformation.businessName'),
+          submissionValidation('businessInformation.businessName'),
+          submissionValidation('businessInformation.clientType'),
+          submissionValidation('businessInformation.legalType'),
+          ]"
+        :loading="loading"
+        @update:selected-value="autoCompleteResult = $event"
+        @update:model-value="validation.business = false"
+        />
+        <cds-inline-loading status="active" v-if="showDetailsLoading">Loading client details...</cds-inline-loading>
+        
+        <cds-inline-notification
+          v-shadow="2"
+          id="bcRegistryDownNotification"
+          v-if="bcRegistryError || (error?.response?.status ?? false)"
+          low-contrast="true"
+          open="true"
+          kind="error"
+          hide-close-button="true"
+          title="BC Registries is down">
+            <span class="body-compact-01"> You'll need to try again later.</span>      
+        </cds-inline-notification>
+
+
+    </data-fetcher>
+
+    <cds-inline-notification
+      v-shadow="2"
+      id="bcRegistrySearchNotification"
+      v-if="!formData.businessInformation.clientType"
+      low-contrast="true"
+      open="true"
+      kind="info"
+      hide-close-button="true"
+      title="If the business does not appear in the list">
+      <div class="cds--inline-notification-content">
+        <ol type="1" class="bulleted-list">
+          <li class="body-compact-01">Ask the applicant if it’s registered with BC Registries</li>
+          <li class="body-compact-01">If it’s not registered, tell them it must be before the application can go ahead</li>
+        </ol>
+      </div>
+    </cds-inline-notification>
+
+    <div v-if="showFields || showOnError" class="read-only-box">
+
+      <cds-inline-notification
+      id="readOnlyNotification"
+      v-shadow="2"
+      v-if="formData.businessInformation.goodStandingInd === 'Y' || showOnError"
+      low-contrast="true"
+      open="true"
+      kind="info"
+      hide-close-button="true"
+      title="Read-only">
+        <div class="body-compact-01">This information is from BC Registries</div>
+      </cds-inline-notification>
+
+      <read-only-component label="Type" id="legalType">    
+        <span class="body-compact-01">{{ legalTypeText }}</span>
+      </read-only-component>
+
+      <hr class="divider" />
+
+      <read-only-component label="Registration number" id="registrationNumber">
+        <template v-slot:icon>
+          <Information16 />
```

Comment:
It seems like this was expected to be a tooltip.
But actually I could only confirm it by using the "navigable form" on Figma.
Its text is: "This is the BC registration number"

Reply:
I asked Marina about this and she said no tooltip, just the icon. I though the same thing.